### ISSUE TITLE
I don't have an email for you

### ISSUE THREAD

**Issue by pwstudent** [bawatin/DjangoTUT#1]
but you should email me at afairley@cs.umb.edu as to your availability for an exam session 12/18/15 from 12-3

**Comment by bawatin** [comment]
Yes I'm avaliable. I have emailed you the same response but I just wanted to make sure. Will the test take place in the same room than the original scheduled test.
